src/MediaDeviceHandler.ts

```diff
@@ -78,6 +81,20 @@ export default class MediaDeviceHandler extends EventEmitter {
 
         await MatrixClientPeg.get().getMediaHandler().setAudioInput(audioDeviceId);
         await MatrixClientPeg.get().getMediaHandler().setVideoInput(videoDeviceId);
+        await MatrixClientPeg.get().getMediaHandler().setAudioSettings(MediaDeviceHandler.loadAudioSettings());
+    }
+
+    private static loadAudioSettings(): AudioSettings {
+        return {
+            autoGainControl: SettingsStore.getValue("webrtc_audio_autoGainControl"),
+            echoCancellation: SettingsStore.getValue("webrtc_audio_echoCancellation"),
+            noiseSuppression: SettingsStore.getValue("webrtc_audio_noiseSuppression"),
+        };
+    }
```

Comment:
If we remove the `audioSettings` member, this can also go

Reply:
Removed.